lib/metaschema-config/category.js

```diff
@@ -58,6 +58,16 @@ const addCategoryData = (type, entity, ms) => {
   return errors;
 };
 
+const checkDetail = category =>
+  ['Catelog', 'Subdivision', 'Hierarchy']
```

Comment:
Also, why not make them lowercase to begin with?

Reply:
So that error message says `can not be divided by Catalog` not `can not be divided by catalog`.